flutter/lib/src/event_processor/url_filter/html_url_filter_event_processor.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:html' as html show window, Window;
+
+import '../../../sentry_flutter.dart';
+import 'url_filter_event_processor.dart';
+// ignore: implementation_imports
+import 'package:sentry/src/utils/regex_utils.dart';
+
+// ignore_for_file: invalid_use_of_internal_member
+
+UrlFilterEventProcessor urlFilterEventProcessor(SentryFlutterOptions options) =>
+    WebUrlFilterEventProcessor(options);
+
+class WebUrlFilterEventProcessor implements UrlFilterEventProcessor {
+  WebUrlFilterEventProcessor(
+    this._options,
+  );
+
+  final SentryFlutterOptions _options;
+
+  @override
+  SentryEvent? apply(SentryEvent event, Hint hint) {
+    final frames = _getStacktraceFrames(event);
+    final lastPath = frames?.first?.absPath;
+
+    if (lastPath == null) {
+      return event;
+    }
```

Comment:
have you tried if this works with a release build? 

you'll also need the sentry_dart_plugin to upload source maps if you haven't